test/functionalTest/cases/3658_env_vars/env_vars.test

```diff
@@ -0,0 +1,134 @@
+# Copyright 2020 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+--NAME--
+broker env vars set on command line
+
+--SHELL-INIT--
+
+--SHELL--
+
+#
+# 01. Get the extended usage from the broker withut any alterations - see default state
+# 02. Set the broker pidfile using env var ORION_PID_PATH=/tmp/cb.pid, start the broker in BG, make sure that /tmp/cb.pid has been created
+#
+
+echo "01. Get the extended usage from the broker withut any alterations - see default state"
+echo "====================================================================================="
+contextBroker -U | cut -c 1-200
+echo
+echo
+
+
+echo "02. Set the broker pidfile using env var ORION_PID_PATH=/tmp/cb.pid, start the broker in BG, make sure that /tmp/cb.pid has been created"
+echo "========================================================================================================================================"
+export ORION_PID_PATH=/tmp/cb.pid
```

Comment:
```suggestion
export ORION_PID_PATH=/tmp/cb3658.pid
```

To make the name even more "random" and avoid possible colisions.

Reply:
Fixed in cde3f08